src/main/org/deidentifier/arx/ARXNaiveBayesConfiguration.java

```diff
@@ -65,6 +67,15 @@ public static ARXNaiveBayesConfiguration create() {
     private ARXNaiveBayesConfiguration(){
         // Empty by design
     }
+    
+    /*
+     * (non-Javadoc)
+     * @see org.deidentifier.arx.ARXClassificationConfiguration#getClassifierName()
+     */
+    @Override
+    public String getClassifierName() {
```

Comment:
Please don't do this. This breaks the strict separation between the packages main and gui. Classes in main must never import classes from the gui package.